### ISSUE TITLE
Using the RStudio Terminal- Lesson 5: Command Line Programs

### ISSUE THREAD

**Issue by C-CAB** [swcarpentry/r-novice-inflammation#487]
[Lesson 5: Command Line Programs](http://swcarpentry.github.io/r-novice-inflammation/05-cmdline/index.html)

Given the current structure of the Programming with R lessons, I wonder if it makes sense to detail out whether students should have a separate terminal running for using the command line or if they can use the terminal integrated within Rstudio. If this is a student's first time with any kind of programming I think sticking to the terminal in Rstudio will feel a lot less daunting as everything feels a little more compact. On the other hand, working directly with the terminal might make them feel more comfortable with it if they decide to work with future lessons focusing specifically on the Unix shell.

What are other thoughts on this?

**Comment by tobyhodges** [comment]
Perhaps a callout box mentioning both options, linking to [a suitable resource for them to learn more about working in the shell](http://swcarpentry.github.io/shell-novice/) and [instructions to access a terminal on their system](http://swcarpentry.github.io/shell-novice/setup.html), with the recommendation that they use RStudio's builtin terminal interface if this is their first time? It would also need to mention that, if using a separate terminal, they'll need to navigate to the correct working directory before the commands in the episode will work.

**Comment by HaoZeke** [comment]
It is my experience that shell novices don't associate any of the R lesson content with the standard terminal, and so this might be more confusing than helpful. A callout sounds like good idea, and this is an important discussion!

**Comment by Bedxxe** [comment]
I agree with the callout option. Also, I think that the time stipulated in this episode is inaccurate. For students, it may take more than 30 minutes to abstract the information of the terminal and complete successfully the exercises. I deem that 30 minutes for teaching and 20 minutes for the exercises could be better. Moreover, I think it would be useful to talk about text editor options that can help them to cope with typing errors.
